be/src/storage/tablet_updates.cpp

```diff
@@ -349,10 +349,19 @@ Status TabletUpdates::_load_from_pb(const TabletUpdatesPB& tablet_updates_pb) {
     }
     del_vector_cardinality_by_rssid.clear();
 
+    // make sure there is no delvec/dcg cache for all rowsets in this TabletUpdates
+    // to avoid the potential unconsistents metadata read.
+    for (auto& [id, rowset] : _rowsets) {
+        _clear_rowset_del_vec_cache(*rowset);
+        _clear_rowset_delta_column_group_cache(*rowset);
+    }
+
     l2.unlock(); // _rowsets_lock
 
     RETURN_IF_ERROR(_load_pending_rowsets());
     StorageEngine::instance()->update_manager()->clear_cached_del_vec(tsids);
```

Comment:
looks like del_vec_cache is cleared multiple times, can we simply this just remove once

Reply:
Updated